### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by johnpapa**

### CODE REVIEW

**Comment by johnpapa** [comment]
@TimFerrell take a look at this and make sure I didnt do anything horrible :) I refactored the code a bit to be what I think is a bit cleaner.

File: tests/unit/toastr-tests.js [hunk 1]
```diff
@@ -75,19 +75,26 @@
...
-        //Act
-        $toast = toastr.info(msg, sampleTitle + '-1');
-        $toast.find('button').focus();
-        toastr.clear($toast);
-        //Assert
-        ok($toast.is(':visible'), 'Focused toast after a clear is visible');
-        //Teardown
-        resetContainer();
-    });
+//    test('clear - after clear toast with focus still appears', 1, function () {
```
Comment (TimFerrell) [inline tests/unit/toastr-tests.js]: Remove commented code blocks.

File: toastr.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /*
  * Toastr
- * Copyright 2012-2014 
+ * Copyright 2012-2014
```
Comment (TimFerrell) [inline toastr.js]: Can we uptick the year?

File: demo.html [hunk 3]
```diff
@@ -193,6 +198,7 @@
 
             return msgs[i];
         };
+
         var getMessageWithClearButton = function (msg) {
             msg = msg ? msg : 'Clear itself?';
             msg += '<br /><br /><button type="button" class="clear">Yes</button>';
```
Comment (TimFerrell) [inline demo.html]: Add class "btn" to keep in line with Bootstrap styling used by other example toasts.

**Comment by TimFerrell** [comment]
Comments made inline.

Overall looks good; tests passed.

File: toastr.js (hunk 2 above)
Comment (johnpapa) [inline toastr.js]: yes, please do. pull this and add that change

File: demo.html (hunk 3 above)
Comment (johnpapa) [inline demo.html]: good one. please do so

**Comment by johnpapa** [comment]
I think we can take this and up everything to 2.1.1 and release on bower, nuget, cdnjs, and github. wanna give it a shot? the changelog.md has the list

**Comment by TimFerrell** [comment]
Made some changes as suggested above.
We also need to update the checklist to push out to npm.

It's been a while since we've pushed up, not entirely sure my memory would be accurate. If you want to take this release and push it to master and and all other sources that's fine by me.

**Comment by johnpapa** [comment]
cant be merged. needs some resolutions. wanna take a stab at that?

**Comment by TimFerrell** [comment]
I'll check it out.

**Comment by TimFerrell** [comment]
Develop merged into master. 
@johnpapa  - Ready for push out to CDNs.

**Pull request closed without merging**